service/history/queues/dlq_writer_test.go

```diff
@@ -172,4 +174,6 @@ func TestDLQWriter_Ok(t *testing.T) {
 	counter, ok := recordings[0].Value.(int64)
 	assert.True(t, ok)
 	assert.Equal(t, int64(1), counter)
+	assert.Len(t, recordings[0].Tags, 1)
+	assert.Equal(t, recordings[0].Tags[metrics.TaskCategoryTagName], task.GetCategory().Name())
```

Comment:
Actually, I think [Equal](https://pkg.go.dev/github.com/stretchr/testify/assert#Equal)'s first parameter is the "expected", not the "actual" (as it is here).